.github/workflows/release.yml

```diff
@@ -9,23 +9,25 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        shell: bash -el {0}
+        shell: bash -l {0}
     container:
-      image: jammont/isofit:latest
+      image: jammont/isofit:amd64
+      options: --user root
 
     steps:
+    - name: Enable ISOFIT Environment
+      run: |
+        echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
```

Comment:
Likely remove this as it will likely conflict with the docker container's built in Conda environment